tests/components/flux_led/test_light.py

```diff
@@ -93,6 +93,52 @@ async def test_light_no_unique_id(hass: HomeAssistant) -> None:
     assert state.state == STATE_ON
 
 
+async def test_light_firmware_old_protocol(hass: HomeAssistant) -> None:
```

Comment:
IMO, this should be parametrized as it's pretty much the same code path for both old & new protocols.

Reply:
3071fe82c3

Reply:
done